### PR TITLE
Update of the generator which remove unneeded extra backlines

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
@gkoz: I have shared feelings about adding empty line between functions declaration: it makes reading easier, but files longer. What do you think?

### CODE REVIEW

**Comment by gkoz** [comment]
I guess considering we'll be putting the docs there, blank lines between functions make sense. But this should probably be better reflected in the notes then. Also make sure you're using `gir` from `master` when making the last version of the PR (i.e. regenerate after the gir PR is merged).

**Comment by GuillaumeGomez** [comment]
Ok, I update the commit message on `gir`.

**Comment by gkoz** [comment]
> Also make sure you're using `gir` from `master` when making the last version of the PR (i.e. regenerate after the gir PR is merged)

This will pick up other changes to gir though making you commit message here incomplete yet again :(

**Comment by GuillaumeGomez** [comment]
That's not a real problem, I'll just regenerate the sources and change the commit message here as well.

**Comment by GuillaumeGomez** [comment]
@gkoz: Updated with the gir master branch.

File: src/auto/actionable.rs [hunk 1]
```diff
@@ -17,19 +17,25 @@ glib_wrapper! {
...
+
     //#[cfg(gtk_3_4)]
     //fn set_action_target_value(&self, target_value: /*Ignored*/&glib::Variant);
+
     #[cfg(gtk_3_4)]
     fn set_detailed_action_name(&self, detailed_action_name: &str);
 }
 
 impl<O: Upcast<Actionable>> ActionableExt for O {
+
```
Comment (gkoz) [inline src/auto/actionable.rs]: Bummer. We've missed this case.

File: src/auto/bin.rs [hunk 2]
```diff
@@ -21,10 +21,10 @@ pub trait BinExt {
 }
 
 impl<O: Upcast<Bin>> BinExt for O {
+
```
Comment (GuillaumeGomez) [inline src/auto/bin.rs]: `impl` case is broken. I fix it.

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by gkoz** [comment]
Yay! Thanks.

**Pull request merged**